ui/webui/src/components/aboutmodal.scss

```diff
@@ -0,0 +1,53 @@
+@import "global-variables";
+
+.pf-c-about-modal-box__strapline {
+    padding-top: 0;
+  }
+  
+  /* Scale down the about box to be a little smaller */
+  @media only screen and (min-width: 992px) {
+    .pf-c-about-modal-box {
+      --pf-c-about-modal-box--lg--MaxWidth: 62rem;
+      --pf-c-about-modal-box--lg--MaxHeight: 38rem;
+      grid-template-rows: 3rem max-content auto;
+      grid-template-columns: 1fr 0fr;
+    }
+  }
+  
+  /* Hide the logo and side graphic */
+  .pf-c-about-modal-box__brand,
+  .pf-c-about-modal-box__hero {
+      display: none;
+  }
+  
+  /* Make the close button more obvious, since we hide the side graphic*/
+  .pf-c-about-modal-box__close .pf-c-button.pf-m-plain {
+    --pf-c-about-modal-box__close--c-button--BackgroundColor: var(--pf-global--BackgroundColor--dark-200);
+    position: absolute;
+  }
+  .pf-c-about-modal-box__close .pf-c-button.pf-m-plain:hover {
+    --pf-c-about-modal-box__close--c-button--BackgroundColor: var(--pf-global--BackgroundColor--dark-400);
+  }
+  
+  #about-modal-flex{
```

Comment:
Please remove all the custom CSS from here and bellow, all can be fixed with using the proper layout components from patternfly.